apps/ios/SimpleXChat/ChatTypes.swift

```diff
@@ -598,7 +635,11 @@ public struct GroupMember: Identifiable, Decodable {
     public var chatViewName: String {
         get {
             let p = memberProfile
-            return p.displayName + (p.fullName == "" || p.fullName == p.displayName ? "" : " / \(p.fullName)")
+            if p.localAlias == "" {
+                return p.displayName + (p.fullName == "" || p.fullName == p.displayName ? "" : " / \(p.fullName)")
+            } else {
+                return p.localAlias
+            }
```

Comment:
```suggestion
            return p.localAlias == ""
                        ? p.displayName + (p.fullName == "" || p.fullName == p.displayName ? "" : " / \(p.fullName)")
                        : p.localAlias
```

Reply:
why it's not using chatViewName here?

Reply:
if you mean profileViewName it's different